src/internal.c

```diff
@@ -15053,6 +15063,15 @@ static word32 QSH_KeyExchangeWrite(WOLFSSL* ssl, byte isServer)
                         return NO_PEER_KEY;
                     }
 
+                #ifdef HAVE_PK_CALLBACKS
+                    if (ssl->ctx->EccSharedSecretCb) {
+                        ret = ssl->ctx->EccSharedSecretCb(ssl, encSecret + OPAQUE8_LEN,
```

Comment:
This return code (ret) also gets stomped on without a check by export_x963() below.